File: c/retangulo.c
#include <stdio.h>
#include <math.h>

int main(){
    double base, altura, area, perimetro, diagonal;

    printf("Base do retangulo: ");
    scanf("%lf", &base);

    printf("Altura do retangulo: ");
    scanf("%lf", &altura);

    area = base * altura;

    printf("AREA = %.4lf\n", area);

    perimetro = 2 * (base + altura);

    printf("PERIMETRO = %.4lf\n", perimetro);

    diagonal = sqrt(base * base + altura * altura);

    printf("DIAGONAL = %.4lf\n", diagonal);

	return 0;
}

File: python/retangulo.py
import math

base: float; altura: float; area: float; perimetro: float; diagonal: float;

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = base * altura
perimetro = 2 * (base + altura)
diagonal = math.sqrt(base * base + altura * altura)

print(f"AREA = {area:.4f}")
print(f"PERIMETRO = {perimetro:.4f}")
print(f"DIAGONAL = {diagonal:.4f}")

